.github/workflows/pull_request.yml

```diff
@@ -74,17 +74,17 @@ jobs:
         run: pipenv run python -m scripts.extract_translation_templates --test
       - name: Python linting
         run: make lint-python
-      - name: Get yarn cache
-        id: get-yarn-cache
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-      - name: Cache yarn modules
+      - name: Get npm cache
+        id: get-npm-cache
+        run: echo "dir=$(npm get cache)" >> $GITHUB_OUTPUT
+      - name: Cache npm modules
         uses: actions/cache@v3
-        id: cache-yarn
+        id: cache-npm
         with:
-          path: ${{ steps.get-yarn-cache.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
-      - name: Install yarn deps
-        run: yarn install
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
```

Comment:
We no longer need a hash etc, caching some be done when setting up node. https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data

This will then be consistent with validator.

Reply:
Just to clarify, would it be appropriate to remove lines 80 through to 85? Or would I also remove the `get npm cache` too?

Reply:
But, I was able to remove the cache-npm and move caching into setup-node which works as GHA works as expected.